frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -528,13 +531,7 @@ export const eventUsageLogic = kea<eventUsageLogicType>({
 
             properties.total_event_actions_count = (properties.events_count || 0) + (properties.actions_count || 0)
 
-            let totalEventActionFilters = 0
-            const entities = (filters.events || []).concat(filters.actions || [])
-            entities.forEach((entity) => {
-                if (entity.properties?.length) {
-                    totalEventActionFilters += entity.properties.length
-                }
-            })
+            const totalEventActionFilters = 0
```

Comment:
why this change? 🤔

Reply:
Ooooo good catch! That shouldn't be there - sloppy commits (had started a refactor and then decided against it)